### DISCUSSION TITLE
cron jobs not triggering

### DISCUSSION THREAD

**Discussion by baburaaj** [YetiForceCompany/YetiForceCRM#16833] [How to]
I installed Yetiforce in my website using softaculous from the cpanel. I see that in the cron tab , there are entries for the cron and I see that in Yetiforce cron is enabled. But it is not getting triggered . Can you help in checking where is the issue

**Comment by mariuszkrzaczkowski** [comment]
https://yetiforce.com/en/knowledge-base/documentation/administrator-documentation/item/enable-cron
